superdesk/default_settings.py

```diff
@@ -756,3 +757,24 @@ def local_to_utc_hour(hour):
 #: .. versionadded:: 2.0
 #:
 KEYWORDS_ADD_MISSING_ON_PUBLISH = False
+
+
+#: Enable archive autocomplete API
+#:
+#: .. versionadded:: 2.0
+#:
+#: It will return suggestions for field from archive api,
+#: only from published items for time defined via
+#: :data:`ARCHIVE_AUTOCOMPLETE_DAYS` and :data:`ARCHIVE_AUTOCOMPLETE_HOURS`.
+#:
+ARCHIVE_AUTOCOMPLETE = False
```

Comment:
Should anything in `default_settings.py` use `env` utility method when defining settings?

Reply:
I don't think so. It make sense for eg db connection which would be different in different environments where you deploy an instance (testing/production), but a feature you either use and have it on everywhere or you don't use it.